### PR TITLE
INT-86 Pass the weight if non-zero and add total weight

### PR DESCRIPTION
**Pull request by JTaylorEight**

### CODE REVIEW

**Review by devin-haslam — APPROVED** [review]
👍 Changes look good

Should we worry about `IsZero` comparing floats?

File: models/api_process_shipment.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ type Shipment struct {
 
 	// Only used for international ground shipments
 	OriginatorName    string
+	TotalWeight       Weight
```
Comment (jonathan-law-pypl) [inline models/api_process_shipment.go]: Just double checking if total weight is used for international only?

**Review by jonathan-law-pypl — COMMENTED** [review]
Please update PR title to be more descriptive than "Int 86", this becomes the commit name

File: models/api_process_shipment.go (hunk 1 above)
Comment (JTaylorEight) [reply models/api_process_shipment.go]: I'll move TotalWeight to match comments

**Comment by JTaylorEight** [comment]
No, that looks into the structure to see if value is zero not unit

**Pull request merged**